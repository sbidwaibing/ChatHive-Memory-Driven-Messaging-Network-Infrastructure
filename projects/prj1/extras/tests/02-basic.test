+ @zdu test #xx #yy
This is a test message
.
? test
.
? test #xx
.
? test #yy #xx
.
+ @user test #zz
Yet another test message
.
? test 5
.
? test #xx #zz
.
? test 1
.
