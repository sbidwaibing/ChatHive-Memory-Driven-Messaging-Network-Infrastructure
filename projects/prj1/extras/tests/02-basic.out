@zdu test #xx #yy
This is a test message
@zdu test #xx #yy
This is a test message
@zdu test #xx #yy
This is a test message
@user test #zz
Yet another test message
@zdu test #xx #yy
This is a test message
@user test #zz
Yet another test message
